core/pom.xml

```diff
@@ -64,6 +64,18 @@
             <scope>provided</scope>
         </dependency>
 
+        <!-- Registry -->
+        <dependency>
+            <groupId>io.apicurio</groupId>
+            <artifactId>apicurio-registry-utils-converter</artifactId>
```

Comment:
We need to review the archetype if we add this in core.

Reply:
An issue has been created for addressing it: https://github.com/apache/camel-kafka-connector/issues/693